tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+import os
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture()
+def test_data_dir():
+    here = Path(os.path.dirname(__file__))
+    return here / 'data'
```

Comment:
Should this assert something? Like whether the data dir exists, rather than returning it.